app/views/editions/secondary_nav_tabs/edit/draft/_transaction.html.erb

```diff
@@ -0,0 +1,85 @@
+<%= render partial: "editions/secondary_nav_tabs/edit/draft/components/title", locals: { edition: @resource } %>
+
+<%= render partial: "editions/secondary_nav_tabs/edit/draft/components/meta_tag_description", locals: { edition: @resource } %>
+
+<%= render "govuk_publishing_components/components/textarea", {
+  label: {
+    heading_size: "s",
+    text: "Introduction",
+  },
+  name: "edition[introduction]",
+  value: edition.introduction,
+  rows: 16,
+  hint: ("Set the scene for the user. What is about to happen? For example, “you will need to fill in a form, print it out and take it to the post office”. Refer to the #{link_to("Govspeak guidance (opens in new tab)", "https://govspeak-preview.publishing.service.gov.uk/guide", target: "_blank", rel: "noopener", class: "govuk-link")}").html_safe,
+} %>
+
+<%= render "govuk_publishing_components/components/radio", {
+  heading: "Start button text",
```

Comment:
We need to add `heading_level: 0` here to prevent the legend text being rendered as a `<h2>`

Reply:
We should also make the same change in `_beta_content_radio.html.erb`, which is not part of this PR.